src/skins/vector/css/vector-web/structures/_RoomSubList.scss

```diff
@@ -33,14 +33,13 @@ limitations under the License.
     font-weight: 600;
     font-size: 12px;
     width: 203px; /* padding + width = LHS Panel width */
-    height: 17px; /* padding + height = 29px, same as mx_RoomSubList_stickyContainer */
+    height: 19px; /* padding + height = 29px, same as mx_RoomSubList_stickyContainer */
```

Comment:
You're replacing the 2px border with an extra 2px of height? What's the reason fir doing so? And could we update the comment too, otherwise it becomes a lie since 19 + (6 * 2) != 29

Reply:
Yep, this seemed the easiest option, without the border or height there would be gaps between the "stuck" headers. Increasing the height is one option, another would have been leaving the border and colouring it the same as the background but that seemed a bit pointless.

I shall comment as such.

Reply:
I realised it should have been 
`/* height + padding = 31px = mx_RoomSubList_label height */`

Reply:
ok, yeah, I think that makes sense!